src/dotnet-install.sh

```diff
@@ -1130,4 +1093,5 @@ else
     say "Binaries of dotnet can be found in $bin_path"
 fi
 
+say "Please check the full list of dependencies in the documentation: https://docs.microsoft.com/en-us/dotnet/core/install"
```

Comment:
Would people clicking the link get lost in the pages before they can reach de "dependencies" section?

Maybe we can prevent that by being a bit more explicit, like:
"To check the full list of dependencies, please go to the following link, select your operating system and check the \"Depedencies\" section: https://docs.microsoft.com/en-us/dotnet/core/install"

I am not sure this is the proper way to escape quotes.

Reply:
I agree we may be more verbose. However, I would like to make a message that looks like a necessary action item. To be seen it also should be either quite short or colored. The script is for developer's use, I hope that they can work with documentation and find information there. I can suggest modifying the message like that:
```
Please check the full list of dependencies for your operating system in the documentation: https://docs.microsoft.com/en-us/dotnet/core/install
```

Reply:
However, I thought that maybe we should explain that the script does not resolve dependencies. So, we anyway need to be verbose. Here is the next proposal, two lines:
```
Please note the script does not resolve dependencies during installation.
To check the list of dependencies, go to https://docs.microsoft.com/en-us/dotnet/core/install, select your operating system, and check the "Dependencies" section.
```